src/seedu/addressbook/data/person/Phone.java

```diff
@@ -35,6 +35,11 @@ public Phone(String phone, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidPhone(String test) {
         return test.matches(PHONE_VALIDATION_REGEX);
     }
+    
+    @Override
```

Comment:
You have not implemented the second part of the question, where you were suppose to print a list of printable objects.